packages/kbn-docs-utils/src/api_docs/README.md

```diff
@@ -0,0 +1,12 @@
+# Autogenerated API documentation
+
+[RFC](../../../rfcs/text/0014_api_documentation.md)
+
+This is an experimental api documentation system that is managed by the Kibana Tech Leads until
```

Comment:
@stacey-gammon Hi! I realize this is, like, 3 weeks old, but...

Temperate check? How has it been? Any flags so far?

Reply:
Hey @goodroot, thanks for asking! It's going well but there are a few issues that I suspect are causing folks to not adopt and use this:

1. Performance. The page may even pop up an error that says  "page failed to respond within x seconds". https://github.com/elastic/elastic-docs/issues/274 

2. Link scroll positions are off and the section is not auto-expanded.  https://github.com/elastic/elastic-docs/issues/262 

3. Usage tracking and analytics: https://github.com/elastic/elastic-docs/issues/309. I'll need this to be able to prove whether these API docs are valuable. If they aren't being used much, I can dig into the why.

Reply:
Very helpful -- thank you! I am gathering very broad information and will circle back. 🕺